service/controller/clusterapi/v17/resources/awsclusterconfig/create.go

```diff
@@ -135,12 +161,28 @@ func (r *Resource) mapClusterToAWSClusterConfig(awsClusterConfig corev1alpha1.AW
 		},
 	}
 
-	// TODO: Workers shall be added when we have better understanding towards template structure.
+	var workers []corev1alpha1.AWSClusterConfigSpecGuestWorker
+	for _, md := range machineDeployments {
+		for i := 0; i < int(md.Status.Replicas); i++ {
```

Comment:
Use live replica count from status because that's the field that keeps up with tenant cluster as well (and therefore provides correct value as basis for scaling ingress PODs etc. in `cluster-operator`).